### PR TITLE
Workshop

### PR DESCRIPTION
**Pull request by annaarnold96**
Did terribly. Had a bit of a breakdown ngl. Couldn't figure out how to turn the list into objects that I could actually use.

### CODE REVIEW

File: cli/pom.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+    <dependency>
+        <groupId>io.github.cdimascio</groupId>
+        <artifactId>java-dotenv</artifactId>
+        <version>5.2.2</version>
+    </dependency>
+
+    <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
```
Comment (pablobgldo) [inline cli/pom.xml:44]: Nice use of dependencies

File: cli/src/main/java/org/example/Main.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ * INSERT INTO users VALUES(1,'Sally_Kunde','Sally','Kunde','Sally.Kunde49@yahoo.com','https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/730.jpg','MNkcfO7qfSqzJl5');
+ * INSERT INTO users VALUES(2,'Ebba.Cole','Ebba','Cole','Ebba.Cole@gmail.com','https://avatars.githubusercontent.com/u/54619199','nz5H7F98ukot7yv');
+ * INSERT INTO users VALUES(3,'Gerson_Barton','Gerson','Barton','Gerson.Barton@gmail.com','https://avatars.githubusercontent.com/u/69465068','OTSg4YbbUQIMZ6P');
+ * **/
+
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
+
+    public class User {
```
Comment (pablobgldo) [inline cli/src/main/java/org/example/Main.java:26]: Maybe you could try create the User class in another file (User.java) and add getters to access them from Main.java?

File: cli/src/main/java/org/example/Main.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        String first_name;
+        String  last_name;
+        String email;
+        String avatar;
+        String password;
+    }
+
+    public static void main(String[] args) {
+
+        Path filePath = Paths.get("reports", "users.json");
```
Comment (pablobgldo) [inline cli/src/main/java/org/example/Main.java:38]: Iconic way of defining both paths of interest - will be replicating this in my project later!

File: cli/src/main/java/org/example/Main.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        User userList = mapper.readValue((JsonParser) lines, User.class);
+
+        System.out.println(userList);
+
+
+        List<String> test = new ArrayList<>();
+
+        test.add("hello");
+
+        Files.write(newFilePath,test);
```
Comment (pablobgldo) [inline cli/src/main/java/org/example/Main.java:55]: I assume it's not finished but you could try write a list of 'SQL Insert statement' strings into the newFilePath and do some logic before to create each string?

**Review by pablobgldo — CHANGES_REQUESTED** [review]
Keep up the good work!! You are so close!

**Review by Charlie-robin — APPROVED** [review]
Good to merge :)

**Pull request closed without merging**